scripts/docstrings_checker_test.py

```diff
@@ -21,6 +21,7 @@
 
 
 class ASTDocStringCheckerTest(unittest.TestCase):
+    """Class for testing the docstrings_checker script."""
```

Comment:
just a confirmation, is it docstring_checker or docstrings_checker? The class name is singular, while the file name is plural. Can we try to be consistent?

Reply:
Done